### ISSUE TITLE
Some Emails are Sent without Email Body

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#7018]
**Describe the bug**

We've seen especially through Fresh-desk that certain automated emails that we are sending through the platform comes without an email body. 

**To Reproduce**

It's random and I haven't found a way to reproduce. 

**Examples**

https://opencollective.freshdesk.com/a/tickets/254973

https://opencollective.freshdesk.com/a/tickets/247034

https://opencollective.freshdesk.com/a/tickets/240246

https://opencollective.freshdesk.com/a/tickets/221612

Generally speaking, you can find many here (some of them are spam users but others are legitimate automated emails that we send); https://opencollective.freshdesk.com/a/search/tickets?term=Not%20given.

**Comment by znarf** [comment]
How the body looks like when inspected on Mailgun?

**Comment by SudharakaP** [comment]
> How the body looks like when inspected on Mailgun?

We don't have the body of emails for these in Mailgun as they keep the email for only 3 days. I'll check periodically and get back when I another one of these occurrences happen.

**Comment by Betree** [comment]
@SudharakaP do you have recent reports of this one?

**Comment by SudharakaP** [comment]
> [@SudharakaP](https://github.com/SudharakaP) do you have recent reports of this one?

Yes this is still happening and you can find all those issues using; https://opencollective.freshdesk.com/a/search/tickets?sortBy=relevance&term=%22Not%20given%22. 

Now some of them are spam I think, but I feel there are legitimate ones too; https://opencollective.freshdesk.com/a/tickets/374084
